package-lock.json

```diff
@@ -1535,7 +1535,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
```

Comment:
Please revert the changes to package-lock.json.

Reply:
Reverted the package-lock back to previous commit.